vignettes/cookbook_metrics.Rmd

```diff
@@ -0,0 +1,203 @@
+
+**Assumption: Market share stays constant ($m(t)$ = 1)**
+
+Due to the lack of quantitative data on the expected market share changes throughout the entire time horizon up to 2050. $m(t)$ is set to 1 for all years. Under the SBTI method for calculating $m(t)$, there will be a higher intensity reduction target in cases where the absolute pathway of the sector exceeds the scenario target. This makes sense. However, applying this at company level is counter-intuitive:
+
+> Companies that decrease their market share would be allowed to have a higher CO2-Intensity than the average market actor. While, companies that are increasing their market share are forced to do more in terms of CO2-Intensity than ones whose market share remains constant. It follows that if a company reaches the targeted CO2-Intensity it would not be allowed to increase its share in the market. This is a desirable outcome.
+
+Under this assumption, our target calculation reduces to: 
+
+$$I^{Target}(t) = \left( d * p (t) \right) + I^{Sc}(2050)$$
+
+**Approximation: Adjust base year scenario emission intensity**
+
+In both the SBTI and the PACTA methodology the target emissions for the sector are taken from climate scenarios. These implement a global economy top-down approach which applies an absolute emissions value in the year 2050 and then converts this to yearly emission intensities. However, there may be discrepancies between the Scenario projected emission intensities, and the bottom-up ABCD emission intensities. To reflect this discrepancy, we adjust the scenario projections by the following factor, 
+
+$$\dfrac{I^{ABCD}(t_0)}{I^{Sc}(t_0)}$$
+yielding the adjusted scenario pathway: 
+
+$$I'^{Sc}(t) = \left(\dfrac{I^{ABCD}(t_0)}{I^{Sc}(t_0)}\right) * I^{Sc}(t)$$
+This yields the final PACTA SDA target equation:
+
+$$I^{Target}(t) = \left( d * p (t) \right) + I'^{Sc}(t)$$
+Note: $d$ and $p(t)$ also must be  re-calculated using this adjusted scenario intensity, $I'^{Sc}$. 
+
+### Calculating SDA Targets
+
+To calculate SDA targets you need to use the `{pacta.loanbook}` package and a number of datasets, including a "matched" dataset (loanbook + asset-level data) that you can get get by following the [Matching Process](cookbook_running_the_analysis.html#matching-process) section of the [Running the Analysis](cookbook_running_the_analysis.html) chapter of the cookbook. The datasets used here are included in the `{pacta.loanbook}` package; they are fake but show how you should structure your own data.
+
+* Load the package.
+
+```{r, message = FALSE}
+library(pacta.loanbook)
+```
+
+* Match the loanbook to asset level data.
+
+```{r, R.options = list(width = 400)}
+loanbook <- loanbook_demo
+abcd <- abcd_demo
+
+matched <- match_name(loanbook = loanbook, abcd = abcd) %>%
+  # WARNING: Remember to validate the output of match_name() before prioritize()
+  prioritize()
+
+matched
+```
+
+* Calculate SDA targets for CO2 emissions intensities:
```

Comment:
SDA targets can be calculated at portfolio level and at company level too, just like market share targets. I suggest we show both, because not doing so for one of the metrics may lead to misunderstandings on the metric's limitations

Reply:
done